packages/angular_devkit/core/src/logger/logger.ts

```diff
@@ -21,6 +21,7 @@ export interface LogEntry extends LoggerMetadata {
 export interface LoggerApi {
   createChild(name: string): Logger;
   log(level: LogLevel, message: string, metadata?: JsonObject): void;
+  next(entry: LogEntry): void;
```

Comment:
It's used in Jobs in a few places. It's because if we only have a `LoggerApi` in and we give a `LoggerApi` that just forwards, we need some method that pass in a LogEntry.

If you have a better idea for a name, let me know.

Reply:
`log`?

Reply:
`log` is already taken, I'd rather not overload it.

Reply:
> Having next in LoggerAPI essentially bypasses the API proper to give access to something not even the base Logger class provides. It bypasses the logging API to instead use it as a Observable<LogEntry> directly. A sort of backdoor I guess.

Typed, and not backdoor; `Logger` already extends `Observable<LogEntry>`. The difference is `LoggerApi` does not.

What I need is a way to connect two loggers through the `LoggerApi` using composition. I could change it to have a `Logger` itself instead, but `LoggerApi` is the more generalized API.

Either I overload `log` to take a `LogEntry` (which is surprisingly complex for subclasses of logger), or I add a new method.

Reply:
@clydin it seems like this last thread that's left unresolved from you. Are you okay with the state of it right now?